internal/commands/app/describe.go

```diff
@@ -0,0 +1,65 @@
+package app
+
+import (
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/terminal"
+	"github.com/spf13/pflag"
+)
+
+// set of supported `app describe` command strings
+const (
+	CommandDescribeUse     = "describe"
+	CommandDescribeDisplay = "app describe"
+)
+
+// set of supported `app describe` command strings
+var (
+	CommandDescribeAliases = []string{}
+)
+
+type describeInputs struct {
+	cli.ProjectInputs
+}
+
+func (i *describeInputs) Resolve(profile *cli.Profile, ui terminal.UI) error {
+	return nil
+}
+
+// CommandDescribe is the `app describe` command
+type CommandDescribe struct {
+	inputs describeInputs
+}
+
+// Flags is the command flags
+func (cmd *CommandDescribe) Flags(fs *pflag.FlagSet) {
+	cmd.inputs.Flags(fs)
+}
+
+// Inputs is the command inputs
+func (cmd *CommandDescribe) Inputs() cli.InputResolver {
+	return &cmd.inputs
+}
+
+// Handler is the command handler
+func (cmd *CommandDescribe) Handler(profile *cli.Profile, ui terminal.UI, clients cli.Clients) error {
+	if cmd.inputs.Project == "" && cmd.inputs.App == "" {
```

Comment:
I wonder if we have the `ProjectInputs.Resolve` in the `describeInputs.Resolve` implementation, if this is necessary anymore?

Reply:
I still don't think we need this bit... `cli.ResolveApp` handles the case where the `filter.GroupID` is blank